scripts/compileClient.sh

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env bash
```

Comment:
I'd put this file in `/erizo_controller/erizoClient` and again, the default should be including the needed libraries. That way Licode would work outside of the box, even if you run gulp directly.

Reply:
yeah it makes sense

Reply:
It makes sense for the current structure, but I'd prefer to move ALL the scripts in the script folder.
I think it's way more clear to the people that approach Licode for the first time